conda/recipes/libcuspatial/meta.yaml

```diff
@@ -57,15 +74,24 @@ outputs:
       run_exports:
         - {{ pin_subpackage("libcuspatial", max_pin="x.x") }}
       ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
```

Comment:
You only need this line. The CUDA 12 compiler doesn’t have anything you need to ignore, and you can drop cupti by passing a CMake flag to disable nvbench cupti support as mentioned above.